From 65bbb75e44c0368bcabbe2b1b428f1809993cdc0 Mon Sep 17 00:00:00 2001
From: Oleg Kovalov <iamolegkovalov@gmail.com>
Date: Mon, 16 Apr 2018 21:01:46 +0200
Subject: [PATCH] fix: t.Fatalf parameters

---
 api_test.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/api_test.go b/api_test.go
index 488f3d1..306c49b 100644
--- a/api_test.go
+++ b/api_test.go
@@ -323,7 +323,7 @@ func TestPopulateProxyWithBadDataShouldReturnError(t *testing.T) {
 
 		for _, p := range proxies {
 			if p.Name == "two" || p.Name == "three" {
-				t.Fatalf("Proxy %s exists, populate did not fail correctly.")
+				t.Fatalf("Proxy %s exists, populate did not fail correctly.", p.Name)
 			}
 		}
 	})
